bikeshed/Spec.py

```diff
@@ -262,11 +260,12 @@ def serialize(self):
     def fixMissingOutputFilename(self, outputFilename):
         if outputFilename is None:
             # More sensible defaults!
-            if self.inputSource.endswith(".bs"):
-                outputFilename = self.inputSource[0:-3] + ".html"
-            elif self.inputSource.endswith(".src.html"):
-                outputFilename = self.inputSource[0:-9] + ".html"
-            elif self.inputSource == "-":
+            if isinstance(self.inputSource, FileInputSource):
```

Comment:
We could move the output filename guessing into the InputSource class, and we could use something other than "-" for URLs, like use the last segment of the URL's path.

Reply:
What would that mean? We do a POST to that URL?

Reply:
No, I was thinking to save it to the current directory, using a name based on the last segment. I'm happy with the '-' default for now.